src/redux/categories/categories.js

```diff
@@ -0,0 +1,25 @@
+const SET_CATEGORIES = 'SET_CATEGORIES';
+
+const setCategories = (categories) => ({
+  type: SET_CATEGORIES,
+  payload: categories,
+});
+
+const initialState = {
+  categories: [],
+  status: 'Under construction',
+};
+
+const categoriesReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case SET_CATEGORIES:
+      return {
+        ...state,
+        categories: action.payload,
+      };
+    default:
+      return state;
+  }
+};
+
+export { SET_CATEGORIES, setCategories, categoriesReducer };
```

Comment:
- [ ] Your implementation of the category reducer and actions do not follow the requirements. You are required to import `createSlice` from the redux toolkit and use it to set up your reducer for the category. Kindly refactor this file to fit this requirement. You can refer to [this documentation](https://redux-toolkit.js.org/api/createSlice).

Reply:
categories.js file refactored to import createSlice from redux toolkit✔